special-pages/pages/new-tab/app/widget-list/WidgetList.js

```diff
@@ -71,13 +70,13 @@ export function WidgetList() {
                  * This section is for elements that the user controls the visibility of
                  */
                 return (
-                    <WidgetVisibilityProvider key={widget.id} visibility={matchingConfig.visibility} id={matchingConfig.id} index={index}>
+                    <WidgetVisibilityProvider key={widget.id} id={widget.id} index={index}>
```

Comment:
For performance reasons, we no longer set the visibility here, since updating it for 1 components caused additional work on sibling elements.